src/LibYear/Output/Json/DisplayVersion.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Text.Json.Serialization;
+using LibYear.Core;
+
+namespace LibYear.Output.Json;
+
+internal sealed record DisplayVersion
+{
+	[JsonPropertyName("versionNumber")]
+	public string VersionNumber { get; init; } = string.Empty;
+	[JsonPropertyName("releaseDate")]
+	public DateTime ReleaseDate { get; init; }
+	public DisplayVersion(Release release)
```

Comment:
Can we just use the underlying `Release` here? Might require a `JsonConverter<PackageVersion>` that returns its `ToString()`

Reply:
Happy to put full release there, but given everything is currently `public`, everything would get included
That would currently look like
```json
{
  "YearsBehind": 0,
  "DaysBehind": 0,
  "Projects": [
    {
      "Project": "test project 1",
      "YearsBehind": 0,
      "Packages": [
        {
          "PackageName": "test1",
          "CurrentVersion": {
            "versionNumber": "1.2.3",
            "releaseDate": "2024-05-30",
            "Release": {
              "Version": {
                "WildcardType": 0,
                "Version": "1.2.3.0",
                "IsLegacyVersion": false,
                "Revision": 0,
                "IsSemVer2": false,
                "OriginalVersion": null,
                "Major": 1,
                "Minor": 2,
                "Patch": 3,
                "ReleaseLabels": [],
                "Release": "",
                "IsPrerelease": false,
                "HasMetadata": false,
                "Metadata": null
              },
              "Date": "2024-05-30",
              "IsPublished": true
            }
          }
        }
      ]
    }
  ]
}
```

Is there anything that would to be removed/hidden/etc

Reply:
Sorry, should have been clearer here. Can we just use more of the existing model objects/classes in the JSON above?
- `Packages` becomes an array of `Result`
- `Release` (of which `Result` has 2, `Installed` and `Latest`) contains a `PackageVersion`
- Could that `PackageVersion` use a `JsonConverter<PackageVersion>` that returns just the `ToString()` interpretation of the version? (I may not have a proper understanding of `JsonConverter<T>`)

So the JSON from above would look like:
```json
{
  "YearsBehind": 0,
  "Projects": [
    {
      "Project": "test project 1",
      "YearsBehind": 0,
      "Packages": [
        {
          "Name": "test1",
          "Installed": {
            "Version": "1.2.3",
            "Date": "2024-05-30",
            "IsPublished": true
          },
          "Latest": {
            "Version": "1.2.3",
            "Date": "2024-05-30",
            "IsPublished": true
          }
        }
      ]
    }
  ]
}
```